.prettierrc.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   singleQuote: true,
-  trailingComma: 'all'
```

Comment:
Not trying to be too meta, but one we release the new package we could use `@side/prettier-config` here as well (or remove it and configure within the package.json like you callout in the README)

Reply:
I'm not sure about the circular nature of the dependencies there, but I think that is a cool idea! Would it be okay to explore that in a follow up PR?

Reply:
I think it's more of a headache to do what you're suggesting Scott. Considering the config isn't that complicated, I'd suggest we stick with just spelling it out in this lib.